livestream-compose-sample/src/main/java/io/getstream/livestream/compose/LivestreamComposer.kt

```diff
@@ -0,0 +1,72 @@
+package io.getstream.livestream.compose
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Send
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.compose.ui.messages.composer.components.MessageInput
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.viewmodel.messages.MessageComposerViewModel
+
+@Composable
+fun LivestreamComposer(
+    composerViewModel: MessageComposerViewModel,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier
+            .fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        MessageInput(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 8.dp)
+                .weight(1f),
```

Comment:
We use the weight here, but don't really have other components in the `Row`, or rather other that use the weight. We should probably add a weight to the `IconButton`, or maybe a fixed size.

Reply:
We do have send button after `input` component in this row 

dint follow this point

Reply:
For now updated this component , let me know if u had some ideas to improve